packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_controller.dart

```diff
@@ -441,39 +441,44 @@ class DeepLinksController extends DisposableController
     }
   }
 
-  Future<List<LinkData>> _validateAndroidDomain(
+  Future<List<LinkData>> _validateDomain(
     List<LinkData> linkdatas,
   ) async {
     final domains = linkdatas
         .where(
-          (linkdata) =>
-              linkdata.os.contains(PlatformOS.android) &&
-              linkdata.domain != null,
+          (linkdata) => linkdata.domain != null,
         )
         .map((linkdata) => linkdata.domain!)
         .toSet()
         .toList();
 
-    late final Map<String, List<DomainError>> domainErrors;
-
+    late final Map<String, List<DomainError>> androidDomainErrors;
+    Map<String, List<DomainError>> iosDomainErrors = {};
     try {
-      final result = await deepLinksServices.validateAndroidDomain(
+      final androidResult = await deepLinksServices.validateAndroidDomain(
         domains: domains,
         applicationId: applicationId,
         localFingerprint: localFingerprint.value,
       );
-      domainErrors = result.domainErrors;
+      androidDomainErrors = androidResult.domainErrors;
       googlePlayFingerprintsAvailability.value =
-          result.googlePlayFingerprintsAvailability;
+          androidResult.googlePlayFingerprintsAvailability;
+      if (FeatureFlags.deepLinkIosCheck)
+        iosDomainErrors = await deepLinksServices.validateIosDomain(
+          domains: domains,
+          applicationId: applicationId,
+          localFingerprint: localFingerprint.value,
+        );
     } catch (_) {
       //TODO(hangyujin): Add more error handling for cases like RPC error and invalid json.
       pagePhase.value = PagePhase.validationErrorPage;
       return linkdatas;
     }
 
     return linkdatas.map((linkdata) {
-      final errors = domainErrors[linkdata.domain];
-      if (errors != null && errors.isNotEmpty) {
+      final errors = (androidDomainErrors[linkdata.domain] ?? []) +
+          (iosDomainErrors[linkdata.domain] ?? []);
```

Comment:
use spread operators instead of the `+` operator:
```
final errors = [
  ...(androidDomainErrors[linkdata.domain] ?? []),
  ...(iosDomainErrors[linkdata.domain] ?? []),
];
```